interface/super/edit_list.php

```diff
@@ -1023,6 +1045,14 @@ function mysubmit() {
                     <select name='list_id' class="form-control select-dropdown"
                             id="list_id">
                         <?php
+                        /*
+                         * Keep proper list name (otherwise list name changes according to
+                         * the options shown on the screen).
+                         */
+                        $list_id_container = trim(strip_tags($_GET["list_id_container"]));
```

Comment:
@yurige-matrix , remove the strip_tags() . I don't see where $list_id is sent to html output in this script, so not needed.

Reply:
Hi @bradymiller , I think it's better to call the "attr" function instead trim and strip_tags, because I will use the $ list_id_container variable (=> $list_id) to query the sql query in line 1328. What you think?

Reply:
hi @yurige-matrix ,
attr() is also not needed, since this variable is getting sql escaped (via binding) in that query.